dashboard/modules/reporter/reporter_agent.py

```diff
@@ -237,8 +254,10 @@ def _get_all_stats(self):
         self._network_stats_hist.append((now, network_stats))
         self._network_stats_hist = self._network_stats_hist[-7:]
         then, prev_network_stats = self._network_stats_hist[0]
-        netstats = ((network_stats[0] - prev_network_stats[0]) / (now - then),
-                    (network_stats[1] - prev_network_stats[1]) / (now - then))
+        net_speed_stats = ((network_stats[0] - prev_network_stats[0]) /
```

Comment:
```suggestion
        network_speed_stats = ((network_stats[0] - prev_network_stats[0]) /
```
The code style for Ray OSS is to use full spelling if possible and try to be consistent with surrounding code. So net -> network

Reply:
To make it easier to read, let's unwind the [0]s and [1]s. 
```
prev_send, prev_recv = prev_network_stats
now_send, now_recv = network_stats
duration = now- then
net_speed_stats = (now_send-prev_send)/duration, (now_recv-prev_recv)/duration
```

Reply:
made changes